README.md

```diff
@@ -2,14 +2,28 @@
 
 [Github Actions](https://developer.github.com/actions/) developed/used by Espressif to help manage GitHub repos.
 
-* `sync_issues_to_jira` performs one way syncing of GitHub issues into a JIRA project.
+### Sync issues to JIRA
 
-# Support and Changes
+`sync_issues_to_jira` performs one way syncing of GitHub issues into a JIRA project.
 
-* Raising issues and sending Pull Requests is very welcome.
+### Upload components
 
-* However, please remember this repository exists to serve Espressif's internal GitHub integration needs. Issues or Pull Requests which aren't useful for those needs may not be addressed or merged. Sorry.
+`upload_components` uploads components from a GitHub repo to [Espressif Component Service](https://components.espressif.com)
 
-# License
+| Input       | Optional | Default                               | Description                                                |
+| ----------- | -------- | ------------------------------------- | ---------------------------------------------------------- |
+| api_token   | ❌       |                                       | API Token for component service                            |
+| directories | ✔        | Repo root                             | Semicolon separated list of directories with components    |
+| name        | ✔ / ❌   |                                       | Name of the component, required if only 1 directory is set |
+| namespace   | ✔        | espressif                             | Namespace on the service for components                    |
```

Comment:
I think we can make this explicit — defaulting to `espressif` makes sense only while the component service is not open to other users.